app/views/users/backup_code_setup/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<% title t('forms.backup_code_regenerate.confirm') %>
+
+<%= image_tag(asset_url('alert/warning-lg.svg'), alt: '', width: 54) %>
+
+<h1 class="h3 mb1 mt3 my0">
+  <%= t('forms.backup_code_regenerate.confirm') %>
+</h1>
+
+<div class="col-2 mb2">
+  <hr class="mt3 mb2 bw4 rounded border-yellow"></hr>
+</div>
+
+<p class="mb4">
+  <%= t('forms.backup_code_regenerate.caution') %>
```

Comment:
ditto re: `==` and `raw`

Reply:
Like above, they don't have any tags or interpolation, so it should be safe to not escape them?

https://github.com/18F/identity-idp/blob/master/config/locales/forms/en.yml#L33-L34
https://github.com/18F/identity-idp/blob/master/config/locales/forms/es.yml#L36-L38
https://github.com/18F/identity-idp/blob/master/config/locales/forms/fr.yml#L36-L38

Reply:
Makes sense! Maybe they used to have something. Thanks for checking